core/src/main/java/com/klaytn/caver/kct/kip17/KIP17.java

```diff
@@ -1142,7 +1280,30 @@ private static SendOptions determineSendOptions(KIP17 kip17, SendOptions sendOpt
             value = sendOptions.getValue();
         }
 
-        newSendOptions = new SendOptions(from, gas, value);
+        if(feeDelegation || sendOptions.isFeeDelegation()) {
```

Comment:
`sendOptions ` that the user sends as parameters of the function should be reflected in higher priority than the default SendOptions(`getDefaultSendOptions()`).

Therefore, even if feeDelegation(from `getDefaultSendOptions()`) is `true`, it should be set `false` if `sendOptions.isFeeDelegation()` is `false`.

The default SendOptions should be the value used if the user has not set sendOptions.